src/plays/sticky-notes/StickyNotes.js

```diff
@@ -0,0 +1,23 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import App from './App';
+
+// WARNING: Do not change the entry componenet name
+function StickyNotes(props) {
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">

```

Comment:
Add an additional class name(unique) and use that in your CSS file to prefix the style names.

Reply:
after i made the changes do i have to make another pr or just have to push the code?

Reply:
No no, you need to push to the same branch. Same PR

Reply:
fixed the issues

Reply:
@Coderamrin Do not add the style `stickyNotes-body` at the top level. As you are changing fonts, it will impact the common header as well.

Add it at the line # 13 along with `play-details-body`. Thanks!